crates/revm/src/evm_impl.rs

```diff
@@ -640,9 +644,12 @@ impl<'a, GSPEC: Spec, DB: Database, const INSPECT: bool> EVMImpl<'a, GSPEC, DB,
             });
         }
 
+        let hash = bytecode.hash_slow();
```

Comment:
On the line 601
You can replace `self.code` with something like
```
         self
            .data
            .journaled_state
            .load_code(address, &mut self.data.db)
            .map_err(|e| ...;
```
This would return an Account that contains both the hash and bytecode.

Reply:
Done in https://github.com/bluealloy/revm/pull/628/commits/f895c57874acd6db2b5877375734d3102a08b4ea, albeit involves a bytecode clone to not use `take_bytecode`.